GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx

```diff
@@ -61,8 +84,42 @@ const RegistrationForm = () => {
         if (e.target.value !== "annet") setDessertInput("");
     } 
 
+    const onSubmit1 = (e : React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        let personsList = Array<PersonType>();
+        persons.forEach(p => {
+            const p1:PersonType = {
+                Wish: p.wish,
+                Age: parseInt(p.age),
+                Gender: p.gender,
+            };
+            personsList.push(p1);
+        });
+
+        let submit:submittype = {
+            Dinner:dinnerRadio,
+            Dessert:dessertRadio,
+            Note:specialNeeds,
+            Event:"JUL2021",
+            Location:location,
+            ContactFullName:contactName,
+            ContactEmail:contactEmail,
+            ContactPhoneNumber: contactPhoneNumber,
+            Institution:"NAV",
+            ReferenceId:pid,
+            FamilyMembers: personsList
+        }
+        fetch('https://localhost:5001/api/recipent', {
```

Comment:
This might need to be a config variable?
We can fix this later and potentially make a component that is responsible for communication with the back end.

Reply:
It should be a config variable yeah, but as you said that could be implemented later